clients/unshaded/src/main/java/tachyon/client/WriteType.java

```diff
@@ -16,32 +16,36 @@
 package tachyon.client;
 
 /**
- * Different write types for a TachyonFile.
+ * Convenience modes for commonly used write types for a TachyonFile.
  *
- * As of 0.8, replaced by {@link TachyonStorageType} and {@link UnderStorageType}
+ * For finer grained control over data storage, advanced users may specify
+ * {@link tachyon.client.TachyonStorageType} and {@link tachyon.client.UnderStorageType}.
  */
-@Deprecated
 public enum WriteType {
   /**
-   * Write the file and must cache it.
+   * Write the file and must cache it. No data will be persisted to the under storage.
    */
   MUST_CACHE(1),
   /**
-   * Write the file and try to cache it.
+   * Write the file and try to cache it. This write type is deprecated and not recommended for
+   * use. Use either MUST_CACHE or CACHE_THROUGH depending on your data persistence requirements.
    */
+  @Deprecated
```

Comment:
Can you also add to the documentation about the version since which it's deprecated?